### DISCUSSION TITLE
curl -X POST -F "file=@D:\BaiduNetdiskDownload\14936228.PDF" http://127.0.0.1:5001/console/api/files/upload?source=datasets {     "code": "not_setup",     "message": "Dify has not been initialized and installed yet. Please proceed with the initialization and installation process first.",     "status": 401 }

### DISCUSSION THREAD

**Discussion by xuboot** [langgenius/dify#4956] [General]
curl -X POST -F "file=@D:\BaiduNetdiskDownload\14936228.PDF" http://127.0.0.1:5001/console/api/files/upload?source=datasets
{
    "code": "not_setup",
    "message": "Dify has not been initialized and installed yet. Please proceed with the initialization and installation process first.",
    "status": 401
}

**Comment by patryk20120** [comment]
Is the Dashboard working correctly? Can you log in and create apps?
